spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -37,6 +42,8 @@
       expect(response).to have_http_status(:ok)
       json_response = parse_response(response)
 
+      course_jsonapi["attributes"]["name"] = "Generated Title"
```

Comment:
not sure whats this mock is trying to tell me...

``` ruby
# actual service code
    def generated_title(subjects)
      return "" if subjects.empty?
```

give it nothing and mock it so something returns? bad bakery?

Reply:
I wanted to write a test that just tested "Was this title being generated correctly", but this was the nicest way I could find :man_shrugging: i'll probably change this now i'm rereading this